src/js/login.js

```diff
@@ -41,12 +74,12 @@ loginForm.addEventListener("submit", function login(event){
 	$.ajax({
 		url: loginEndPoint,
 		dataType: "jsonp",
-		jsonpCallback: "p",
 		jsonp: "callback",
 		crossDomain: true,
 		success: function (response) {
 			if(response.accepted){
-				accessToken = response.accessTken;
+                headingEmail.innerHTML="Logged in as " + email;
+				accessToken = response.access_token;
```

Comment:
@ms10398 We should consider neglecting `codacy` in this case since `access_token` cannot be written in `camelCase`. Do you have any other ideas? Please tell @stealthanthrax

Reply:
Yes I have seen it cant be changed this looks fine to me